### DISCUSSION TITLE
What's next for gGoods

### DISCUSSION THREAD

**Discussion by xavier506** [eoscostarica/gGoods#223] [General]
- New avatar categories: Humans, Robots, Zombies, Objects, etc
- Improve Security
- Dark Mode and other UI enhancements
- Language Translations
- 3D preview of the assets
- Subscription payments through Google Pay and other payment gateways
- Royalty payments to the community in perpetuity for secondary market sales
- Allowing secondary market sales
- Allowing “My Own NFT Design”
- Ability to create an NFT on behalf of an organization
- NFT filters and categories

**Comment by xavier506** [comment]
- Attaching sound to an NFT

**Comment by xavier506** [comment]
@edgarithm  Lets include `dogs` as one of the new categories we want to create see issue #193

**Comment by murillojorge** [comment]
Allowing the use of eosio wallets and donating with crypto.
